### PR TITLE
Complete the task 1a

### PR DESCRIPTION
**Pull request by aurya-dev**

### CODE REVIEW

**Review by matkatmusic — APPROVED** [review]

**Review by matkatmusic — APPROVED** [review]

File: main.cpp [hunk 1]
```diff
@@ -109,7 +109,19 @@ Thing 4) Gas Grill
     3) turn of the flame
  */
 
-
+/*
+Thing 10) Wireless Headphone
+5 properties:
+    1) Speaker
+    2) Over Ear
```
Comment (matkatmusic) [inline main.cpp]: `Over Ear` can be represented as a Primitive (`bool`), not as an Object.  Choose an Object that has 5 properties of its own and 3 abilities.
- [x] revise

Comment (matkatmusic) [inline main.cpp]: This isn't a very interesting object.  If I were you, i would choose something like `Synthesizer`, which has Oscillators, Filters, Amps, Envelopes, Effects, Modulations, etc, which all have their own set of properties and functions.
- [x] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 1 above)
Comment (aurya-dev) [reply main.cpp]: Ok I unterstand. I meant this here: https://images-na.ssl-images-amazon.com/images/I/51AoAGTUs6L._AC_.jpg
What's the correct english word for this?

Comment (aurya-dev) [reply main.cpp]: You're right. I changed it to a drum maschine ;)

**Review by matkatmusic — APPROVED** [review]

**Review by matkatmusic — APPROVED** [review]

**Review by matkatmusic — CHANGES_REQUESTED** [review]
It seems as though every member variable and function argument is an `int`
Can you use some `float, double, bool, char, std::string` types instead?

**Comment by aurya-dev** [comment]
> It seems as though every member variable and function argument is an `int`
> Can you use some `float, double, bool, char, std::string` types instead?

Yes, that's boring. I added some different types.

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**